backend/fms_core/template_prefiller/prefiller.py

```diff
@@ -23,9 +23,13 @@ def PrefillTemplate(template_path, template_info, queryset):
     for sheet_name, sheet_dict in position_dict.items():
         current_sheet = workbook[sheet_name]
         for i, entry in enumerate(queryset.values(*filter(lambda x: x is not None, sheet_dict["queryset_column_list"]))):
-            for prefill_sheet_name, template_column, queryset_column, _ in template_info["prefill info"]:
```

Comment:
A function inside `template_info`?

Reply:
power of the interpreted language ... The select case used to not exist in python because you can just make a dictionary with functions in it ... Now they decided to add lambda and all kind of redundant stuff much to my dislike.